atomic_defi_design/qml/Wallet/SendModal.qml

```diff
@@ -249,7 +269,9 @@ BasicModal {
             DefaultButton {
                 Layout.alignment: Qt.AlignRight
                 text: qsTr("Fix")
-                onClicked: input_address.field.text = API.app.to_eth_checksum_qt(input_address.field.text.toLowerCase())
+                onClicked: {
+                    api_wallet_page.convert_address(input_address.field.text, address_data.to_address_format)
```

Comment:
This is good but we still need to listen to the update of the converted_address string and assign it to the input text once is changing